### PR TITLE
Added Nearest Airports to search bar

### PR DESCRIPTION
**Pull request by AJWurts**
#18  Mostly complete. Needs some additional bug testing, and the amount of items displays should be limited to 10 or so.

### CODE REVIEW

**Comment by AJWurts** [comment]
#13 completed

File: backend/addsclient.js [hunk 1]
```diff
@@ -3,48 +3,79 @@ const xml2js = require('xml2js');
 
 // Client for the NWS ADDS service
 const AddsClient = {
-  baseUrl: 'https://aviationweather.gov/adds/dataserver_current/httpparam?requestType=retrieve&format=xml&',
```
Comment (speshak) [inline backend/addsclient.js]: Looks like the bulk of this file's changes are whitespace.  Is there a preferred indent/line ending standard?

File: backend/server.js [hunk 2]
```diff
@@ -17,81 +17,120 @@ app.use(cors());
 
 // App routing to get newestMetar for given airport ident
 app.get('/api/newestMetar/:ident', (req, res, next) => {
-  let airportLetters = airportData.resolveIdent(req.params.ident);
```
Comment (speshak) [inline backend/server.js]: Same whitespace question.

File: react-app/src/components/SearchBox.jsx [hunk 3]
```diff
@@ -26,19 +26,41 @@ class SearchBox extends Component {
...
         </button>
+        {nearestAirports ?
+          <div>
+            <span style={{ display: 'inline-block', fontSize: '22px', margin: '0px 8px', verticalAlign: 'middle', color: 'white' }}>
+              Nearest Airports
+            </span>
+
+            {nearestAirports.map((airport, index) => {
+              return <button key={index} onClick={() => this.onAirportClick(airport)} style={{ verticalAlign: 'middle', padding: '5px 10px', marginTop: "5px" }} className='button'>
+                {airport.airport}: {airport.name.toLowerCase() + ", " + airport.state}
```
Comment (speshak) [inline react-app/src/components/SearchBox.jsx]: I feel like this should be
`{airport.name.toLowerCase()}, {airport.state}`

File: backend/addsclient.js (hunk 1 above)
Comment (AJWurts) [reply backend/addsclient.js]: My original JS linter broke, so I went to a new one. The new standard is 4 spaces, however most files don't reflect that

**Comment by AJWurts** [comment]
All changes are reflected in #33, therefore not necessary.

**Pull request closed without merging**